plugin/ViewFieldModifier/ViewFieldModifierHandler.php

```diff
@@ -108,6 +108,9 @@ public function getAllAPIFields(): array
 		}
 
 		$fields = array_values(array_unique($apiFields));
+		if (($key = array_search('vermarktungsstatus', $fields)) !== false) {
```

Comment:
I am confused. If it appears here, that would be a logic error. Can this actually happen?